src/modules/interactive.js

```diff
@@ -0,0 +1,33 @@
+import { updateLocalStorage, displayTasks, taskList } from './addRemoveEdit.js';
+
+// function to make use of checkbox
+const checkBox = (checkBoxID) => {
+  const indexID = checkBoxID.getAttribute('id');
+  const inputTask = document.querySelectorAll('.taska');
+  if (checkBoxID.checked) {
+    inputTask[indexID - 1].style.textDecoration = 'line-through';
+    taskList[indexID - 1].completed = true;
+  } else {
+    inputTask[indexID - 1].style.textDecoration = 'none';
+    taskList[indexID - 1].completed = false;
+  }
+  updateLocalStorage();
+};
+
+// To clear completed tasks
+const clearButton = () => {
+  for (let i = taskList.length - 1; i >= 0; i -= 1) {
+    if (taskList[i].completed) {
+      taskList.splice(i, 1);
+    }
+  }
```

Comment:
Nice project 😄! As requested in the project requirements:
![image](https://user-images.githubusercontent.com/42799579/199863090-30eccc3c-1ab4-45a6-88d0-deed67623ff3.png)

You need to use the `filter()` method, not `splice()` 😉. Please update this function to use the requested method.
> Here is the [documentation](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter) of the filter method, it may help you 😃.